.ci/script.sh

```diff
@@ -47,6 +47,12 @@ else
   make -C kremlin/kremlib/dist/generic -f Makefile.basic -j
   make -j -k
   popd
+
+  # Test 32-bit build; Tests don't work here yet.
```

Comment:
I'm under the impression this will install 32-bit built ocaml bindings for all x64 architectures, possibly leading to less test coverage... can we make this a separate line in travis?

Reply:
Sure, will do